modules/auxiliary/scanner/rdp/rdp_scanner.rb

```diff
@@ -0,0 +1,104 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => 'Identify endpoints speaking the Remote Desktop Protocol (RDP)',
+        'Description'    => %q(
+          This module attempts to connect to the specified Remote Desktop Protocol port
+          and determines if it speaks RDP.
```

Comment:
Can you add something here (or possibly in the doc file) about why you would turn on `CredSSP` and `EarlyUser`?